clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/EventualServerStoreProxy.java

```diff
@@ -111,11 +160,14 @@ public Chain get(long key) throws TimeoutException {
   @Override
   public void append(final long key, final ByteBuffer payLoad) throws TimeoutException {
     delegate.append(key, payLoad);
+    incrementInvalidationsCounter(key);
   }
 
   @Override
   public Chain getAndAppend(final long key, final ByteBuffer payLoad) throws TimeoutException {
-    return delegate.getAndAppend(key, payLoad);
+    Chain chain = delegate.getAndAppend(key, payLoad);
```

Comment:
nope. we have wait for the call to return from server before that call be accounted for